src/js/tabs/pid_tuning.js

```diff
@@ -1313,14 +1370,32 @@ TABS.pid_tuning.initialize = function (callback) {
             updatePidDisplay();
         });
 
-        $('#resetProfile').on('click', function(){
+        $('#resetProfile').on('click', function() {
             self.updating = true;
-            MSP.promise(MSPCodes.MSP_SET_RESET_CURR_PID).then(function () {
-                self.refresh(function () {
+
+            function refresh () {
+                self.refresh(() => {
                     self.updating = false;
 
                     GUI.log(i18n.getMessage('pidTuningProfileReset'));
                 });
+            }
+
+            MSP.promise(MSPCodes.MSP_SET_RESET_CURR_PID).then(() => {
+                if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+                    TuningSliders.resetDefault();
+                    MSP.promise(MSPCodes.MSP_SET_TUNING_SLIDERS, mspHelper.crunch(MSPCodes.MSP_SET_TUNING_SLIDERS)).then(() => {
```

Comment:
This whole process seems to be unneeded, as `MSP_SET_RESET_CURR_PID` already applies the simplified settings for the PID profile being reset: https://github.com/betaflight/betaflight/blob/415d4db5aa355064c49ae0d8cfe4e8b6081d3c35/src/main/flight/pid.c#L230

Reply:
Fixed.